content/_data/changelogs/lts.yml

```diff
@@ -547,6 +547,42 @@
         Prevent possible <code>NullPointerException</code> when listing remote directories using the <code>FilePath#list()</code> and <code>FilePath#listDirectories()</code> APIs.
       pull: 2914
       issue: 44942
+- version: "2.60.3"
+  date: 2017-08-17
+  changes:
+    - type: bug
+      message: >
+        Contributions to the PATH environment variable could result in malformed values on agents on a platform different from master's.
+      issue: 14807
+      pull: 2936
+    - type: bug
+      message: >
+        Robustness improvements related to agent connections.
+      references:
+        - issue: 43496
+        - issue: 38527
+      pull: 2922
+      # also pull: 2923
+    - type: bug
+      message: >
+        JNLP for launching agents now requests Java 8.
```

Comment:
I would link the Java 8 upgrade announcement again. We still receive bugs from those who does not read docs